samples/deployment/test-network/docker-compose.yml

```diff
@@ -3,16 +3,13 @@
 #
 # SPDX-License-Identifier: Apache-2.0
 
-version: '3'
-
 services:
   #  org1
   ecc.peer0.org1.example.com:
-    container_name: ${CC_ID}.peer0.org1.example.com
-    hostname: ${CC_ID}.peer0.org1.example.com
+    container_name: peer0org1_${CC_ID}_ccaas
     image: fpc/fpc-${CC_ID}${HW_EXTENSION:-}:${FPC_VERSION}
     environment:
-      - CHAINCODE_SERVER_ADDRESS=${CC_ID}.peer0.org1.example.com:9999
+      - CHAINCODE_SERVER_ADDRESS=0.0.0.0:9999
```

Comment:
This does not look right. As described [here](https://github.com/hyperledger/fabric-samples/blob/50b69f61579f2f764ca94ab9624d21c51b7812bf/asset-transfer-basic/chaincode-external/chaincode.env#L1), this should be the service endpoint, but it looks more like the listening address.

Reply:
it is the actual listening address. So this change just simplifies the compose file.